// ngx-flash-messages/lib-dist/flash-messages.module.js
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { FlashMessagesService } from './flash-messages.service';
import { FlashMessagesComponent } from './flash-messages.component';
export { FlashMessagesService } from './flash-messages.service';
export { FlashMessagesComponent } from './flash-messages.component';
var FlashMessagesModule = (function () {
    function FlashMessagesModule() {
    }
    return FlashMessagesModule;
}());
export { FlashMessagesModule };
FlashMessagesModule.decorators = [
    { type: NgModule, args: [{
                imports: [
                    CommonModule
                ],
                providers: [
                    FlashMessagesService,
                ],
                declarations: [
                    FlashMessagesComponent,
                ],
                exports: [
                    FlashMessagesComponent,
                ]
            },] },
];
/** @nocollapse */
FlashMessagesModule.ctorParameters = function () { return []; };
